src/libraries/Microsoft.PowerFx.Core/Localization/Span.cs

```diff
@@ -11,14 +11,24 @@
 
 namespace Microsoft.PowerFx.Core.Localization
 {
+    /// <summary>
+    /// Span in the text formula.
+    /// </summary>
     [TransportType(TransportKind.ByValue)]
```

Comment:
>TransportType [](http://example.com/codeflow?start=5&length=13)

TransportType classes are brittle ... there's a custom codegen step that runs against them, reflects over it, and may do weird things based on public vs. internal.  
Definitely double check the changes with PAClient.

Reply:
I only changed public constructors to internal as McCall suggested. Other changes include adding only comments. I did try to compile with PAClient and there seem to be no problem. Should I do any further tests?

Reply:
TransportType uses reflection, so it may actually care about the public vs. internal ctor. Not sure.. It's brittle.

Reply:
I changed the generator there to look at internals as well when we were doing this migration initially, so it should be safe. In general for transport, if it built, you should be fine. But it's something we'll be aware of when this is checked in.

Reply:
Great, thanks!